module/extension/Director/src/Director.hpp

```diff
@@ -29,6 +31,10 @@
 namespace module::extension {
 
     class Director : public NUClear::Reactor {
+    public:
+        /// A task list is just a list of tasks
```

Comment:
A redundant statement is redundant

Reply:
This is true, this statement used to read `a task pack is just a list of tasks`

Reply:
It also has 3 / 👀

Reply:
> It also has 3 /

@LachlanCourt  A comment that starts with `///` or `/**` is a documenting comment, programs like doxygen will treat this as a brief description.

Reply:
Ohhhh I see I see I've only ever seen it as /**

Reply:
Remove this or reword it. This comment really adds nothing

Reply:
I've been trying to document all the variables/types I make so I needed a comment here. I will make a better one